charts/kof-collectors/templates/opentelemetry/node-collector.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   mode: daemonset
+  securityContext:
```

Comment:
We need to pass this from values, by default security context shouldn't be with root permissions